app/services/idv/actions/verify_document_action.rb

```diff
@@ -40,34 +40,27 @@ def enqueue_job
         verify_document_capture_session.requested_at = Time.zone.now
         verify_document_capture_session.create_doc_auth_session
 
-        callback_url = Rails.application.routes.url_helpers.document_proof_result_url(
-          result_id: verify_document_capture_session.result_id,
+        applicant = image_params.to_h.slice(
```

Comment:
can we call this something besides `applicant` ? I'd expect to have like first name, last name, etc etc in an `applicant`

Reply:
yep, changed to `document_attributes` in 45deaf60b, but open to better names